src/domain/collaboration/callout/callout.resolver.mutations.ts

```diff
@@ -225,6 +233,31 @@ export class CalloutResolverMutations {
     return result;
   }
 
+  @UseGuards(GraphqlGuard)
+  @Mutation(() => ICallout, {
+    description: 'Update the visibility of the specified Callout.',
+  })
+  @Profiling.api
+  async updateCalloutPublisher(
+    @CurrentUser() agentInfo: AgentInfo,
+    @Args('calloutData') calloutData: UpdateCalloutPublisherInput
```

Comment:
If this mutation needs to change only the publisher then we need an appropriate dto for only sending the publisherID.

Reply:
No, one mutation to set either.